app/code/core/Mage/CatalogIndex/Model/Data/Configurable.php

```diff
@@ -34,13 +34,13 @@ class Mage_CatalogIndex_Model_Data_Configurable extends Mage_CatalogIndex_Model_
     /**
      * Defines when product type has children
      *
-     * @var boolean
+     * @var array<int, bool>
```

Comment:
Not a blocker, but I don't think PHPStorm recognizes this syntax. However, it would recognize `int[]|bool[]`.

Reply:
I thought it works too ... however , changed it ...